routes.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"encoding/json"
+	"github.com/gorilla/mux"
+	"log"
+	"net/http"
+)
+
+func searchByStandId(w http.ResponseWriter, r *http.Request) {
+	pathParams := mux.Vars(r)
+	w.Header().Set("Content-Type", "application/json")
+	if val, ok := pathParams["standId"]; ok {
```

Comment:
To avoid so much indentation, you could evaluate the negative case and return if that is true:

```
standId, ok := pathParams["standId"]
if !ok {
    w.WriteHeader(http.StatusBadRequest)
    // also a message here
    return
}
```

Since there is no valid id on the request, the appropriate thing to do would let the user that the request it made is invalid with a message explaining briefly why it's invalid